lib/hash-algorithms.h

```diff
@@ -15,6 +15,12 @@
    | Authors: Derick Rethans <derick@derickrethans.nl>                    |
    +----------------------------------------------------------------------+
  */
+
+// Added to support uint32_t on linux because it is defined in stdint.h
+#include "php.h"
+#include "zend.h"
+#include "zend_API.h"
```

Comment:
And as you say, the files in ```lib/``` should not include any PHP related includes. The best way would be to check for ```stdint.h``` existence in ```config.m4```, or just always include it. QuickHash has been designed to only work on Linux anyway.